pom.xml

```diff
@@ -504,6 +509,42 @@
 					<goals>deploy</goals>
 				</configuration>
 			</plugin>
+
+			<!-- Aggregate JavaDoc for use with autoCompleteResponder -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-javadoc-plugin</artifactId>
+				<version>3.1.0</version>
+				<extensions>true</extensions>
+				<executions>
+					<execution>
+						<phase>generate-resources</phase>
+						<goals>
+							<goal>javadoc</goal>
+						</goals>
+					</execution>
+				</executions>
+
+				<configuration>
```

Comment:
Can this configuration be included inside the `execution` tag above to ensure it is not applied in the execution defined in the release profile?